src/components/@shared/Price/Conversion.tsx

```diff
@@ -31,15 +31,15 @@ export default function Conversion({
     if (
       !prices ||
       !price ||
-      price === '0' ||
+      price === 0 ||
```

Comment:
![Screenshot 2022-09-17 at 10 49 47](https://user-images.githubusercontent.com/23175624/190846496-6214319c-95e6-4963-b850-38b22acf0fb3.png)

this line is not necessary anymore :)

Reply:
good catch, thanks 😊